server/tso/allocator_manager.go

```diff
@@ -245,6 +245,29 @@ func (am *AllocatorManager) GetDCLocationInfo(dcLocation string) (DCLocationInfo
 	return infoPtr.clone(), true
 }
 
+// CleanUpDCLocation cleans up certain server's DCLocationInfo
+func (am *AllocatorManager) CleanUpDCLocation() error {
+	am.mu.Lock()
+	defer am.mu.Unlock()
```

Comment:
Why do we need the lock here? It seems that there is no access to the `am.mu` inside the function.